src/util.c

```diff
@@ -520,3 +521,89 @@ int syncRename(const char *oldname, const char *newname)
     fsyncDir(oldname);
     return RR_OK;
 }
+
+int findTimeoutIndex(RaftRedisCommand *cmd)
```

Comment:
I'm not sure this function should be under `utils`. For now, it's relevant only for blocking commands. suggest moving it to `blocked.c`, rename it so it will be easy to understand that this function handles blocking commands only, and abort for unknown commands.